tc/sgx/trusted_worker_manager/enclave/iohandler.edl

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+enclave {
+
+    trusted {
+        // Each enclave interface file needs to have atleast one public ecall.
+        // hence below dummy function is added
+        public void ecall_dummy(); // Non root ECALL
```

Comment:
What does "Non root ECALL" mean ?

Reply:
Non root ecall a.k.a private ecall are the enclave calls which can only be called from particular function(s) defined as part of untrusted block (more restricted than public ecall). 
In this case, since its a dummy ecall we don't need to specify non-root ecall. I will remove the comment.